src/sass/user-profile.scss

```diff
@@ -55,4 +55,52 @@
       margin-right: 5px;
     }
   }
+  
+  .saved-form-information {
+    margin: 0;
+    
+    div {
```

Comment:
Can we give this div a class and use that in the styles?

Reply:
Good idea.